### PR TITLE
Moved content items between packs

### PR DESCRIPTION
**Pull request by IkaDemisto**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold 

## Description
- Deprecated `SetSeverityByScore` and moved to deprecated pack.
- Deprecated `BlockIP ` and moved to deprecated pack.
- Deprecated `DataDomainReputation ` and moved to deprecated pack.
- Deprecated `DefaultIncidentClassifier ` and moved to deprecated pack.
- Moved `StaticAnalyze ` to `D2` pack.

### CODE REVIEW

**Comment by yuvalbenshalom** [comment]
@IkaDemisto just one note about the PR summery: you did not move emailFieldTriggered to D2 pack. not that I think you should. dod you mean something else?

File: Packs/CommonScripts/Scripts/script-emailFieldTriggered.yml [hunk 1]
```diff
@@ -14,11 +14,14 @@ script: |-
   if (email) {
       var body = 'Hello,\nField ' + args.name + ' was changed from ' + args.old + ' to ' + args.new + ' in incident ' + incidents[0].name + '.\n--DBot';
       var subject = 'Field ' + args.name + ' was triggered';
-      executeCommand('SendEmail', {'to':email, 'subject':subject, 'body':body});
+      executeCommand('send-mail', {'to':email, 'subject':subject, 'body':body});
```
Comment (yuvalbenshalom) [inline Packs/CommonScripts/Scripts/script-emailFieldTriggered.yml:17]: did you test that it works?

**Comment by IkaDemisto** [comment]
> @IkaDemisto just one note about the PR summery: you did not move emailFieldTriggered to D2 pack. not that I think you should. did you mean something else?

Moved `StaticAnalyze` to D2 pack. Updated the summary.

File: Packs/CommonScripts/Scripts/script-emailFieldTriggered.yml (hunk 1 above)
Comment (IkaDemisto) [reply Packs/CommonScripts/Scripts/script-emailFieldTriggered.yml:17]: @yuvalbenshalom seems to work. A lot of scripts are executing `send-mail` command, so don't think it should be a problem.

File: Packs/D2/pack_metadata.json [hunk 2]
```diff
@@ -1,20 +1,19 @@
...
-    "updated": "2020-05-14T14:23:06Z",
-    "beta": false,
-    "deprecated": false,
-    "useCases": [],
-    "keywords": [],
-    "dependencies": {}
+  "name": "D2",
+  "description": "D2 Agent pack",
+  "support": "xsoar",
+  "currentVersion": "1.0.0",
```
Comment (anara123) [inline Packs/D2/pack_metadata.json:5]: The indentation should be 4 spaces.
And why the version is not `1.0.1`?

Comment (IkaDemisto) [reply Packs/D2/pack_metadata.json:5]: As long as we are moving items between packs, better to stay with initial version. It is causing issue in later step

**Comment by yuvalbenshalom** [comment]
@IkaDemisto please resolve the conflict

**Comment by IkaDemisto** [comment]
@yuvalbenshalom done

**Pull request merged**